9 Лаба

SkryabinaPolina opened the pull request:
9 Лаба

SkryabinaPolina committed:
9 Лаба 

9 Лаба

NemirovichMark commented:
Здравствуйте, Полина.

Это, конечно, интересно и приятнее визуально, чем консоль, но это не 9-ая лабораторная. Это ближе к 10й в лабораторном практикуме.

Посмотрите файл с заданием к 9й лабораторной в дискорде в канале с материалами курса.

На доработку.

SkryabinaPolina commented:
А можете пожалуйста куда-то сюда прикрепить этот файл, меня нет в дискорде

NemirovichMark commented:
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/4973d4cf-be5f-4cfb-aa57-ce24a3aead4f)

SkryabinaPolina committed:
Add files via upload

SkryabinaPolina committed:
9 лаба

SkryabinaPolina committed:
8 Лаба

8 лаба

SkryabinaPolina committed:
Add files via upload

SkryabinaPolina committed:
10 лаба

NemirovichMark commented:
Здравствуйте, Полина.
8ю принял.
9ю нет.
1й номер. Отдельно эти файлы не запускаются, попробовал в старый архив вставить этот файл, ошибки тоже выдает. Сделать по аналогии со 2м и 3м.
2й
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/62381694-b569-4a9e-ae42-922060432562)
3й номер
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/5c19213c-5365-4ed1-9f01-e7d143106d73)

10ю жду от вашего тимлида.

На доработку.

SkryabinaPolina committed:
9 laba

SkryabinaPolina committed:
10 laba

NemirovichMark commented:
Здравствуйте, Полина.

По 9й лабораторной:
Нельзя сериализовывать интерфейсы в XML, потому что у них нет конструкторов
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/f1ab877c-9a3c-4d36-8a5a-ff398e8ac315)

По 10й лабораторной. Уже лучше. 
Классы для сериализации ещё выделить в отдельный файл.
Диаграмм должно быть 3: классов, процессов и состояний. 
См. 5ю лекцию, как они должны выглядеть (со стрелочками). Ещё в дискорде в общем чате пример диаграммы состояний присылал.

На доработку.

SkryabinaPolina committed:
10 laba

SkryabinaPolina committed:
9 laba 9.2 9.3 Uml

SkryabinaPolina committed:
9 Лаба 

Извините пожалуйста, я случайно прикрепила не то и только сейчас заметила... Проверьте пожалуйста

NemirovichMark commented:
Здравствуйте, Полина.
Диаграмма процессов в 10м нужна.
9й все также не работает 3й номер (из-за недоступности некоторых полей)
![image](https://github.com/NemirovichMark/BBI-23-3/assets/112826355/b30c06e6-a665-4c57-a789-6da8a3d26d33)
